package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "express-generator",
   "description": "Express' application generator",
-  "version": "4.13.1",
+  "version": "4.14.0",
```

Comment:
> Are you not able to remove the version bump from the pull request?

I updated it on my local repo a minute or so after your comment, it's just not pushed.

> There is not even a 4.14 line of Express yet ...

I was not aware about them being needing to be in sync, thanks for letting me know.